### PR TITLE
Should send initial authz resources when reconnect

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
@Okabe-Rintarou-0 Can you review it, since you fixed workload reconnect

**Comment by Okabe-Rintarou-0** [comment]
lgtm

**Comment by hzxuzhonghu** [comment]
cc @supercharge-xsy

File: pkg/controller/xdstest/fake_xdsserver.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func (f *XDSServer) DeltaAggregatedResources(server discoveryv3.AggregatedDiscov
 			return nil
 		case resp := <-f.DeltaResponses:
 			numberOfSends++
-			log.Infof("sending delta response from mock: %v", numberOfSends)
+			log.Infof("========sending delta response from mock: %v", numberOfSends)
```
Comment (nlgwcy) [inline pkg/controller/xdstest/fake_xdsserver.go:83]: maybe no need to modify..

Comment (hzxuzhonghu) [reply pkg/controller/xdstest/fake_xdsserver.go:83]: Yes, will revert

**Pull request merged**